python/lsst/display/matplotlib/matplotlib.py

```diff
@@ -273,14 +394,13 @@ def format_coord(x, y, wcs=self._wcs, x0=self._xy0[0], y0=self._xy0[1],
             return msg
 
         ax.format_coord = format_coord
-        # Stop images from reporting their value as we've already printed it
-        # nicely
-        from matplotlib.image import AxesImage
-        for a in ax.mouseover_set:
-            if isinstance(a, AxesImage):
-                a.get_cursor_data = lambda ev: None  # disabled
-
-        self._figure.tight_layout()
+        # Stop images from reporting their value as we've already
+        # printed it nicely
+        for a in ax.get_images():
+            a.get_cursor_data = lambda ev: None  # disabled
+
+        if False:
```

Comment:
If we are keeping this block in but disabled may be explain in what context we should use `True` here?

Reply:
I changed the `if False` block to a comment on why I didn't use tight_layout()